Features2/tests/features/editContact.feature

```diff
@@ -0,0 +1,23 @@
+Feature:Edit Contact
+    As a user
+    I want to edit the saved contact sometimes
+    so that I can update the contacts when some changes are required
+     Background:
+     Given the user has added following contacts
+        | name | Email | Phone | Contact Type |
+        | abcde | abcde@gmail.com | 00000000 | Professional |
+        | xyz | xyz@gmail.com | 111111111 | Personal |
+        | example |example@gmail.com | 22222222222| Professional |
+        | example2 | example2@gmail.com |3333333333|Personal|
+        | abcd1 |sabhdb@gmail.com | 9876666 | Personal |
+        |  cvbn |cvbn@gmail.com | 090099090 | Professional |
+        | mnbv | mnvb@gmail.com | 098876677 | Professional |
+
+     #scenario created   
+    Scenario:Edit the saved contact
+     When the  user clicks the Edit button
+     And  the user  edit contact dashboard opens
+     And the user changes the required info
+     And the user clicks the update contact button
+     Then the  contact details will be edited
```

Comment:
you have created so many users and this scenario does not specify which user information to edit. Please make related one.